connectors/jetty-http2-connector/src/main/java11/org/glassfish/jersey/jetty/http2/connector/JettyHttp2ClientSupplier.java

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
java11 folder does not seem necessary. It can be put to other in java folder. That would make the build (pom file) simpler.

Reply:
I don't think it will work as expected. Because in the case of the `JettyHttp2ClientSupplier.java` in the separate src folder the resulting jar will look like:

```META-INF/versions/<JDK-version>/<package>/JettyHttp2ClientSupplier.java```

 and for related JDK a proper version will be picked up. However, if the `JettyHttp2ClientSupplier.java` would be in the main src folder it will be located directly in the compiled classes folder `org.glassfish.jersey.jetty.http2.container` (without version separation), and JDK probably won't be able to overwrite it by picking another `JettyHttp2ClientSupplier.java` for a particular JDK version (17+ in our case)

Reply:
actually there is:

https://github.com/eclipse-ee4j/jersey/tree/2.x/core-common/src/main/java8/org/glassfish/jersey/internal/jsr166

Reply:
Yeah, I've got why it shall be here - if we compile 2 similar classes within the same cycle. The compile phase will fail. It is required to exclude from compiling either one or another version of the same class file. It's not possible to compile both files with the same name in a similar compile phase.

Reply:
In case of JDK 17+ and no version separation we will have 2 similar class names in one compile phase.

Reply:
actually, the resulting jar `jersey-jetty-http2-connector.jar` does not have the `versions/11` folder as well, only `versions/17` but sources structure shall be kept for compilation reasons.